app/views/shared/_navbar.html.erb

```diff
@@ -1,12 +1,12 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
   <div class="container-fluid">
     <%= link_to offers_path, class: "navbar-brand" do %>
-      <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
+      <%= image_tag "Chef2Go.png", class: "avatar" %>
     <% end %>
 
     <form class="d-flex align-items-center flex-nowrap" action="/offers" method="get">
 
-    <input class="form-control mr-sm-2" name ="query" type="search" placeholder="Find a cooking class" aria-label="Search">
+    <input class="form-control mr-sm-2" name ="query" type="search" placeholder="Find a chef" aria-label="Search">
```

Comment:
We could go with a more generic message here, to indicate that a user can search for a chef or a cuisine or a food.

Reply:
The space is very small so we cannot add a list. If you have any suggestions, you can write here.  Anyways, it is a very small detail.

Reply:
Not suggesting a list. I meant a more generic message like 'What are you looking for?' or 'Find your next culinary experience'.. Something along those lines. It is a small detail but one that appears on every screen of the website.

We could merge this for now, then discuss and change on Saturday.